src/main/java/school/hei/haapi/service/utils/DataFormatterUtils.java

```diff
@@ -11,6 +12,12 @@
 public class DataFormatterUtils {
   private DataFormatterUtils() {}
 
+  public static String toLocalDateReadable(LocalDate localDate) {
+    DateTimeFormatter pattern = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+    String stringLocalDate = localDate.format(pattern);
+    return stringLocalDate;
```

Comment:
return the localDate.format()

also, the method name is formatDate, and take a format in param